### ISSUE TITLE
DUP-Admin -> FAQ page can be accessed by non system admin users

### ISSUE THREAD

**Issue by manuji** [bcgov/parks-reso-admin#335]

**Comment by JLWade** [comment]
@manuji yes that's correct - FAQ link should be hidden to non-sys admin users

**Comment by JLWade** [comment]
suggest moving back to to-do to resolve :)

**Comment by manuji** [comment]
@Christopher-walsh22 , assigning this back to you to update the navigation panel and the Home page cards.

**Comment by manuji** [comment]
Tested on TEST: **Passed**
* Verified that this issue has been fixed.
